runtime/common/src/apis.rs

```diff
@@ -268,7 +269,7 @@ macro_rules! impl_runtime_apis_plus_common {
 					} else {
 						None
 					};
-
+					let is_transactional = false;
```

Comment:
Create is not a transaction ?

Reply:
No, both `call` and `create` runtime api method calls result in non-transactional evm executions.